include/secp256k1_musig.h

```diff
@@ -358,13 +162,13 @@ SECP256K1_API int secp256k1_musig_session_combine_nonces(
  *  Out:   out32: pointer to a 32-byte array to store the serialized signature
  *  In:      sig: pointer to the signature
  */
-SECP256K1_API int secp256k1_musig_partial_signature_serialize(
+SECP256K1_API int secp256k1_musig_partial_sig_serialize(
     const secp256k1_context* ctx,
     unsigned char *out32,
-    const secp256k1_musig_partial_signature* sig
+    const secp256k1_musig_partial_sig* sig
 ) SECP256K1_ARG_NONNULL(1) SECP256K1_ARG_NONNULL(2) SECP256K1_ARG_NONNULL(3);
 
-/** Parse and verify a MuSig partial signature.
+/** Parse a MuSig partial signature.
```

Comment:
Should this be consistent with `partial_sig_serialize` and `musig_partial_sig` about whether it mentions adaptor signature or not?

Reply:
Removed the notion of adaptor signature entirely because I dislike the term and it's not well defined. Either it's a pre-signature, or a signature.